src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -0,0 +1,328 @@
+
+Here's an example of a decoded SXG file:
+
+```html
+format version: 1b3
+request:
+  method: GET
+  uri: https://example.org/
+  headers:
+response:
+  status: 200
+  headers:
+    Cache-Control: max-age=604800
+    Digest: mi-sha256-03=kcwVP6aOwYmA/j9JbUU0GbuiZdnjaBVB/1ag6miNUMY=
+    Expires: Mon, 24 Aug 2020 16:08:24 GMT
+    Content-Type: text/html; charset=UTF-8
+    Content-Encoding: mi-sha256-03
+    Date: Mon, 17 Aug 2020 16:08:24 GMT
+    Vary: Accept-Encoding
+signature: 
+    label;cert-sha256=*ViFgi0WfQ+NotPJf8PBo2T5dEuZ13NdZefPybXq/HhE=*;
+    cert-url="https://test.web.app/ViFgi0WfQ-NotPJf8PBo2T5dEuZ13NdZefPybXq_HhE";
+    date=1597680503;expires=1598285303;integrity="digest/mi-sha256-03";sig=*MEUCIQD5VqojZ1ujXXQaBt1CPKgJxuJTvFlIGLgkyNkC6d7LdAIgQUQ8lC4eaoxBjcVNKLrbS9kRMoCHKG67MweqNXy6wJg=*;
+    validity-url="https://example.org/webpkg/validity"
+header integrity: sha256-Gl9bFHnNvHppKsv+bFEZwlYbbJ4vyf4MnaMMvTitTGQ=
+
+The exchange has a valid signature.
+payload [1256 bytes]:
+<!doctype html>
+<html>
+<head>
+    <title>SXG example</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
+    <style type="text/css">
+    body {
+        background-color: #f0f0f2;
+        margin: 0;
+        padding: 0;
+    }
+    </style>
+</head>
+<body>
+<div>
+    <h1>Hello</h1>
+</div>
+</body>
+</html>
+```
+
+The `expires` parameter in the signature indicates a SXG's expiration date. A
+SXG may be valid for at most 7 days. If the expiration date of an SXG is more
+than 7 days in the future, the browser will reject it. Find more information on
+the signature header in the [Signed HTTP Exchanges
+spec](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#section-3.1).
+
+
+### Web Packaging
+
+SXGs are a part of the broader [Web
+Packaging](https://github.com/WICG/webpackage) spec proposal family. In addition
+to SXGs, the other major component of the Web Packaging spec is [Web Bundles](/web-bundles/)
+("bundled HTTP exchanges"). Web Bundles are a collection of HTTP resources and
+the metadata necessary to interpret the bundle.
+
+The relationship between SXGs and Web Bundles is a common point of confusion.
+SXGs and Web Bundles are two distinct technologies that don't depend on each
+other—Web Bundles can be used with both signed and unsigned exchanges. The common
+goal advanced by both SXGs and Web Bundles is the creation of a "web packaging"
+format that allows sites to be shared in their entirety for offline consumption.
+
+SXGs are the first part of the Web Packaging spec that Chromium-based browsers will implement.
+
+## How to use SXGs
+
+### Loading SXGs
+
+Initially, the primary use case of SXGs will likely be as a delivery mechanism
+for a page's main document. For this use case, a SXG could be referenced using
+the `<link>` or `<a>` tags, as well as the [`Link`
+header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link).
```

Comment:
Do we mean this content to an explanation of how SXG works, or is this actually supposed to be the instructional content? Because the section heading is "How to use SXGs" yet it's currently phrased as more of a conceptual explainer. E.g. stuff like "The primary use case of SXGs will likely be as a delivery mechanism for a page's main document" doesn't sound like we're giving instructions to the reader.

If we're intending this to be a how-to guide, we should rephrase all of these "how-to" sections to speak directly to the reader. Something like this:

"To use SXG to deliver a page's main document:

[code samples]

Although it's theoretically possible to use `<script>` or `<img>` tags, there isn't much tooling support for this use case yet…"

CC @mihajlija